molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/RestControllerV2.java

```diff
@@ -150,10 +152,10 @@ public RestControllerV2(DataService dataService, MolgenisPermissionService permi
 	{
 		if (molgenisVersion == null) throw new IllegalArgumentException("molgenisVersion is null");
 		if (molgenisBuildDate == null) throw new IllegalArgumentException("molgenisBuildDate is null");
-		molgenisBuildDate = molgenisBuildDate.equals("${maven.build.timestamp}") ?
-				new SimpleDateFormat("yyyy-MM-dd HH:mm").format(new java.util.Date())
-						+ " by IntelliJ" : molgenisBuildDate;
-
+		if (molgenisBuildDate.equals("${maven.build.timestamp}"))
+		{
+			molgenisBuildDate = DateTimeFormatter.ofLocalizedDateTime(MEDIUM).format(now()) + " by IntelliJ";
+		}
```

Comment:
An existing workaround to show some kind of build timestamp when IntelliJ rather than maven has built the code. This updates it to java 8 time classes.